### DISCUSSION TITLE
Git for Windows v2.47.1.windows.1

### DISCUSSION THREAD

**Discussion by gitforwindowshelper** [git-for-windows/git#5276] [Announcements]
Changes since Git for Windows v2.47.0(2) (October 22nd 2024)

This release comes with the first early native [support of Windows/ARM64](https://github.com/git-for-windows/git/issues/3107), ready for testing. Please report any issues!

### New Features

* Comes with [Git v2.47.1](https://github.com/git/git/blob/v2.47.1/Documentation/RelNotes/2.47.1.txt).
* Comes with [cURL v8.11.0](https://curl.se/changes.html#8_11_0).
* Comes with [Git LFS v3.6.0](https://github.com/git-lfs/git-lfs/releases/tag/v3.6.0).

### Bug Fixes

* Due to a bug introduced in the v2.47 cycle, [the installer showed an empty "experimental options" page](https://github.com/git-for-windows/git/issues/5231), which was [fixed](https://github.com/git-for-windows/build-extra/pull/578).
* A potential crash in Git Bash on Insider versions of Windows/ARM64 [was fixed](https://github.com/git-for-windows/msys2-runtime/pull/76).
* On Windows/ARM64, running the 64-bit version of Git for Windows could infrequently cause deadlocked threads (see e.g. [this report](https://github.com/msys2/msys2-autobuild/issues/62) or [this one](https://inbox.sourceware.org/cygwin-developers/78f294de-4c94-242a-722e-fd98e51edff9@jdrake.com/)), [which was addressed](https://github.com/git-for-windows/msys2-runtime/pull/73).

Filename | SHA-256
-------- | -------
Git-2.47.1-64-bit.exe | 25527923debc06515b3016f2d6bca0820656e8281a23be2f43bfb658bd5dda70
Git-2.47.1-32-bit.exe | 121744b5d804f839d7f6e66e594ccafbe345418f0d02a696f6cec866f6eb1e62
PortableGit-2.47.1-64-bit.7z.exe | 4f3f21f4effcb659566883ee1ed3ae403e5b3d7a0699cee455f6cd765e1ac39c
PortableGit-2.47.1-32-bit.7z.exe | 7d022cdf6a79b0206db4274231f51bee271001d8c88fe2e25d5ff81cc0535b80
MinGit-2.47.1-64-bit.zip | 50b04b55425b5c465d076cdb184f63a0cd0f86f6ec8bb4d5860114a713d2c29a
MinGit-2.47.1-32-bit.zip | 3c0fa6e3096c2304f8c6af9cb6bb35623b2e615771cf7be45f9632af83bd9864
MinGit-2.47.1-busybox-64-bit.zip | 7b26cc61866c5c0ec050797f57837bf064eacd3aaa87ae316e96c6936235cf29
MinGit-2.47.1-busybox-32-bit.zip | c5ee149a781c4672537926f967614e9e33233b8fd08811ce967b5f535ce77620
Git-2.47.1-64-bit.tar.bz2 | 45d2bdc96712b0ab785c307461429cf5bd744227108805edaaec7517432db54c
Git-2.47.1-32-bit.tar.bz2 | 99a91af7a6a7a8791ebede7c0fd5339084853ec1a3c4f9dffdb91145ce934c1e
Git-2.47.1-arm64.tar.bz2 | bc7fbbdfbc54d4a245fc0c18f88c2a20f82ec60345879f2fdc2e7f48d100bd61
Git-2.47.1-arm64.exe | 63950d69998ca184b0ade0389a0e0b50e62f4a1ea8da9752449193c9dcda569f
MinGit-2.47.1-arm64.zip | fc5747e187a70147404a94da104dc9f6005a3d45a78a56dbfa132075ad4a45e4
PortableGit-2.47.1-arm64.7z.exe | d366f44ef2b65e11f7b5a1430ae43aceb5f7c640150f325fa4b767f6da472845

<hr /><em>This discussion was created from the release <a href='https://github.com/git-for-windows/git/releases/tag/v2.47.1.windows.1'>Git for Windows v2.47.1.windows.1</a>.</em>

**Comment by dscho** [comment]
I need to clarify a detail about the Git for Windows/ARM64 part: The `git.exe` included in Git-2.47.1-arm64.exe is native, but for technical reasons the included Git Bash is an x64 executable that runs under emulation.

**Comment by bzanin-wdc** [comment]
The link to the git-lfs 3.6.0 release tag is broken due to an absent leading `v`.  It is currently..:
```
https://github.com/git-lfs/git-lfs/releases/tag/3.6.0
```
...but it should be..:
```
https://github.com/git-lfs/git-lfs/releases/tag/v3.6.0
```

**Reply by rimrul** [reply]
The link has now been updated.

**Reply by dscho** [reply]
↪https://github.com/git-for-windows/git-for-windows-automation/pull/103
